docker-compose.yml

```
@@ -0,0 +1,194 @@
+      SERVER_REWRITEBASEPATH: 'false'
+      KIBANA_INDEX: .dashboards
+      XPACK_MONITORING_UI_CONTAINER_ELASTICSEARCH_ENABLED: 'true'
+    networks:
+      - gn-cloud-network
+
+  database:
+    image: mdillon/postgis
+    environment:
+      POSTGRES_USER: "${MDDB_DBNAME}"
+      POSTGRES_PASSWORD: "${MDDB_USERNAME}"
+      POSTGRES_DB: "${MDDB_PASSWORD}"
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+    networks:
+      - gn-cloud-network
+    ports:
+      # expose for local profile without overlapping with a running default postgres port in the host
+      - "55432:5432"
+
+  # Event broker for spring-cloud-bus event messaging
+  rabbitmq:
+    image: rabbitmq:3-management
+    restart: always
+    environment:
+      RABBITMQ_DEFAULT_USER: "guest"
+      RABBITMQ_DEFAULT_PASS: "guest"
+    volumes:
+      - rabbitmq_data:/var/lib/rabbitmq
+    networks:
+      - gn-cloud-network
+    ports:
+      - "5672:5672"
+      - "15672:15672"
+
+  # Eureka service discovery. This is a Discovery First Bootstrap configuration.
+  # The discovery service is the only fixed entry point.
+  # Browse to http://localhost:8761 to check all services are registered.
+  discovery:
+    image: gn-cloud-discovery:${TAG}
+    environment:
+      SERVER_PORT: 8761
+      EUREKA_INSTANCE_HOSTNAME: discovery
+    ports:
+      - 8761:8761 # for development, so services can be run from localhost and find the discovery service running on docker
+    networks:
+      - gn-cloud-network
+#    healthcheck:
+#      test: ["CMD", "curl", "-f", "-m", "1", "http://localhost:8761/actuator/health"]
+#      interval: 1s
+#      timeout: 1s
+#      retries: 15
+
+  # Spring Cloud Config service, provides centralized configuration to all
+  # microservices. Being a Discovery First Bootstrap configuration, it'll
+  # register itself with the Eureka discovery service and can be scaled
+  config:
+    image: gn-cloud-configuring:${TAG}
+    depends_on:
+      - discovery
+    environment:
+      EUREKA_SERVER_URL: ${EUREKA_SERVER_URL}
+      RABBITMQ_HOST: rabbitmq
+      CONFIG_PATH: ./config
+    networks:
+      - gn-cloud-network
+    volumes:
+      - ./config:/workspace/config
+    ports:
+      - 9999:8080
+
+  auth:
+    image: gn-cloud-authorizing:${TAG}
+    depends_on:
+      - config
+      - rabbitmq
+    environment:
+      EUREKA_SERVER_URL: ${EUREKA_SERVER_URL}
+      RABBITMQ_HOST: rabbitmq
+    networks:
+      - gn-cloud-network
+
+  gateway:
+    image: gn-cloud-gateway:${TAG}
+    depends_on:
+      - config
+      - rabbitmq
+    environment:
+      EUREKA_SERVER_URL: ${EUREKA_SERVER_URL}
+      RABBITMQ_HOST: rabbitmq
+    networks:
+      - gn-cloud-network
+    ports:
+      - 9900:8080
+
+  search:
+    image: gn-cloud-searching:${TAG}
+    depends_on:
+      - config
+      - rabbitmq
+    environment:
+      EUREKA_SERVER_URL: ${EUREKA_SERVER_URL}
+      RABBITMQ_HOST: rabbitmq
+    networks:
+      - gn-cloud-network
+
```

Indexing is missing right?

```suggestion
  index:
    image: gn-cloud-indexing:${TAG}
    depends_on:
      - config
      - rabbitmq
    environment:
      EUREKA_SERVER_URL: ${EUREKA_SERVER_URL}
      RABBITMQ_HOST: rabbitmq
    networks:
      - gn-cloud-network
```